.github/workflows/tests.yml

```diff
@@ -0,0 +1,53 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    name: ${{ matrix.ruby }}
+    runs-on: ubuntu-latest
+    env:
+      COVERAGE_RUBY_VERSION: 2.6
+    strategy:
+      matrix:
+        ruby:
+          - 2.4
+          - 2.5
+          - 2.6
+          - 2.7
+          - truffleruby
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Setup Ruby ${{ matrix.ruby }}
+        uses: eregon/use-ruby-action@master
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          architecture: 'x64'
```

Comment:
`architecture` is unused (also on `setup-ruby`, it's a [bug of their README](https://github.com/actions/setup-ruby/pull/27))

Reply:
Thanks for letting me know! :)